noir-projects/aztec-nr/aztec/src/keys/getters.nr

```diff
@@ -55,25 +74,26 @@ fn get_master_key(
 fn fetch_key_from_registry(
     context: &mut PrivateContext,
     key_index: Field,
-    address: AztecAddress
+    address: AztecAddress,
+    header: Header
 ) -> GrumpkinPoint {
     let x_coordinate_map_slot = key_index * 2 + 1;
     let y_coordinate_map_slot = x_coordinate_map_slot + 1;
     let x_coordinate_derived_slot = derive_storage_slot_in_map(x_coordinate_map_slot, address);
     let y_coordinate_derived_slot = derive_storage_slot_in_map(y_coordinate_map_slot, address);
 
     let x_coordinate_registry: SharedMutablePrivateGetter<Field, DELAY> = SharedMutablePrivateGetter::new(
-        *context,
+        context,
         AztecAddress::from_field(CANONICAL_KEY_REGISTRY_ADDRESS),
         x_coordinate_derived_slot
     );
     let y_coordinate_registry: SharedMutablePrivateGetter<Field, DELAY> = SharedMutablePrivateGetter::new(
-        *context,
+        context,
         AztecAddress::from_field(CANONICAL_KEY_REGISTRY_ADDRESS),
         y_coordinate_derived_slot
     );
-    let x_coordinate = x_coordinate_registry.get_current_value_in_private();
-    let y_coordinate = y_coordinate_registry.get_current_value_in_private();
+    let x_coordinate = x_coordinate_registry.get_value_in_private(header);
+    let y_coordinate = y_coordinate_registry.get_value_in_private(header);
 
     GrumpkinPoint::new(x_coordinate, y_coordinate)
 }
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 114]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Unrelated to this pr, but this one is still slightly painful in my eyes as it is just calling the oracle so we have just "hidden" that we are doing an oracle call and not a constrained call. Think it would be better if we deal with this as an oracle directly to not mix it up, but should likely be done as part of #6543<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/6656?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->